Formula/dosbox-staging.rb

```diff
@@ -24,16 +23,11 @@ class DosboxStaging < Formula
   depends_on "sdl2_net"
 
   def install
-    args = %W[
-      --prefix=#{prefix}
-      --disable-dependency-tracking
-      --disable-sdltest
-      --enable-core-inline
-    ]
-
-    system "./autogen.sh"
-    system "./configure", *args
-    system "make", "install"
+    mkdir "build" do
+      system "meson", *std_meson_args, "-Duse_mt32emu=false", ".."
```

Comment:
What happens when we don't set `use_mt32emu`?

Reply:
it would error out missing runtime dependency `mt32emu`
```
==> meson --prefix=/usr/local/Cellar/dosbox-staging/0.77.0 --libdir=/usr/local/Cellar/dosbox-
Last 15 lines from /Users/rchen/Library/Logs/Homebrew/dosbox-staging/01.meson:
Run-time dependency opusfile found: YES 0.12
Dependency threads found: YES unknown (cached)
Run-time dependency sdl2 found: YES 2.0.14
Run-time dependency sdl2_net found: YES 2.0.1
Run-time dependency gl found: YES
Run-time dependency fluidsynth found: YES 2.1.8
Found CMake: /usr/local/opt/cmake/bin/cmake (3.20.5)
Run-time dependency mt32emu found: NO (tried pkgconfig, framework and cmake)
Not looking for a fallback subproject for the dependency mt32emu because:
Use of fallback dependencies is disabled.
Message: You can disable this dependency with: -Duse_mt32emu=false

../meson.build:221:2: ERROR: Dependency 'mt32emu' not found and fallback is disabled
```